app/app.js

```diff
@@ -0,0 +1,56 @@
+import {App, IonicApp, Platform} from 'ionic/ionic';
+
+import {HelloIonicPage} from './pages/hello-ionic/hello-ionic';
+import {ListPage} from './pages/list/list';
+
+@App({
```

Comment:
Decorators, like @App and @Page, allow us to add metadata (extra information) to our components, and are supplied just above the class definition. Decorators are provided by TypeScript, not by the standard ES6 specification.

By using the @App decorator, we are declaring this component as the root component of the application. We also supply a templateUrl so that the component knows which file to use for its view (you can also define the template inline by using template instead of templateUrl).